automated-comments/go/two_fer/wrong_package_comment.md

```diff
@@ -0,0 +1 @@
+The comment describing the package `twofer` isn't quite in the standard format. The section on comments below may help you fix this.
```

Comment:
The package comment for `twofer` does not follow the Go convention. See the section on comments below.